Tests/SonarScanner.MSBuild.TFS.Tests/BuildVNextCoverageReportProcessorTests.cs

```diff
@@ -35,6 +35,21 @@ public class BuildVNextCoverageReportProcessorTests
     {
         public TestContext TestContext { get; set; }
 
+        private const string TRX_PAYLOAD = "<?xml version=\"1.0\" encoding=\"utf-8\" ?>" +
```

Comment:
Only legacy languages like Java need to create multi-line strings in such a tedious fashion ;-)

In C# you can use the literal string syntax e.g.
```
var text = @"a
b
c
d
";
```
That's how XML strings are embedded [elsewhere](https://github.com/SonarSource/sonar-scanner-msbuild/blob/5c23a7da9171e90a1970a31507dce3da3e8ee094/Tests/SonarScanner.MSBuild.PreProcessor.Tests/SonarWebServiceTest.cs#L222) in the tests.

Reply:
Ok, thanks, never used it that way ! It's corrected.